benchto-driver/src/test/java/com/teradata/benchto/driver/loader/BenchmarkLoaderTest.java

```diff
@@ -122,6 +128,56 @@ public void shouldLoadSimpleBenchmark()
     }
 
     @Test
+    public void shouldLoadAllBenchmarks()
+            throws IOException
+    {
+        // Given no withActiveBenchmarks(...)
+        // When
+        List<Benchmark> benchmarks = loader.loadBenchmarks("sequenceId");
+        // Then
+        Set<String> loadedNames = benchmarks.stream()
+                .map(Benchmark::getName)
+                .collect(toSet());
+
+        assertThat(loadedNames)
+                .contains("concurrent-benchmark", "simple-benchmark");
+    }
+
+    @Test
+    public void shouldLoadBenchmarksFromFileSystem()
```

Comment:
Maybe add classpath test, e.g: `shouldLoadBenchmarksFromClasspath`? I am not sure if that is trivial though. What about multiple JARs?